src/scripts/convert_lovd2.php

```diff
@@ -0,0 +1,1312 @@
+                }
+                if ($aIndividual['edited_by'] === false) {
+                    // No curator ID was available, set submtiter ID.
+                    $aIndividual['edited_by'] = lovd_convertUserID($sSubmitterID, 'submitter');
+                }
+                $aIndividuals[$sLOVD2IndividualID] = $aIndividual;
+
+                // Create screening record.
+                $aScreening = lovd_getRecordForHeaders($aOutputHeaders['screening'], $aRecord,
+                    $aSections['screening']);
+                $nScreeningID = lovd_autoIncScreeningID();
+                $aScreening['id'] = $nScreeningID;
+                $aScreening['individualid'] = $aIndividual['id'];
+                $aScreenings[$sLOVD2IndividualID] = $aScreening;
+
+                // Create screening2gene record.
+                $aScreening2Genes[] = array('screeningid' => $nScreeningID,
+                                            'geneid' => $zTranscript['geneid']);
+
+                // Create phenotype record.
+                $aPhenotype = lovd_getRecordForHeaders($aOutputHeaders['phenotype'], $aRecord,
+                    $aSections['phenotype']);
+                $bEmptyPheno = true;
+                foreach ($aPhenotype as $k => $v) {
+                    $bEmptyPheno = $bEmptyPheno &&
+                        (in_array($k, array('id', 'diseaseid', 'individualid')) || lovd_empty($v));
+                }
+                if (!$bEmptyPheno) {
+                    // Skip phenotype because there is no data in phenotype
+                    // record except for ID fields.
+                    $aPhenotype['id'] = lovd_autoIncPhenotypeID();
+                    $aPhenotype['diseaseid'] = $aDisease['id'];
+                    $aPhenotype['individualid'] = $aIndividual['id'];
+                    $aPhenotypes[$sLOVD2IndividualID] = $aPhenotype;
+                }
+
+                // Create individuals2diseases record.
+                $aIndividuals2Disease = lovd_getRecordForHeaders($aOutputHeaders['i2d'], $aRecord);
+                $aIndividuals2Disease['individualid'] = $aIndividual['id'];
+                $aIndividuals2Disease['diseaseid'] = $aDisease['id'];
+                $aIndividuals2Diseases[] = $aIndividuals2Disease;
+            }
+
+            // Create screening2variant record.
+            $aScreening2Variants[] = array(
+                'screeningid' => $aScreenings[$sLOVD2IndividualID]['id'],
+                'variantid' => $aRecord[array_search('ID_variantid_', $aInputHeaders)]);
+        }
+
+        // Create VOG/VOT records
+        $aVOGRecord = lovd_getRecordForHeaders($aOutputHeaders['vog'], $aRecord,
+            $aSections['vog']);
+        $aVOGRecord['chromosome'] = $zTranscript['chromosome'];
+        if ($aVOGRecord['edited_by'] === false) {
+            // No curator ID was available, set submtiter ID.
+            $aVOGRecord['edited_by'] = lovd_convertUserID($sSubmitterID, 'submitter');
+        }
```

Comment:
What if the data simply had never been edited? The `edited_by` would be null, and should remain `null`...